Libs/Common/Shapeworks.h

```diff
@@ -42,6 +46,10 @@ using IdentityTransform  = itk::IdentityTransform<double, 3>;
 using TransformPtr       = GenericTransform::Pointer;
 TransformPtr createTransform(const Matrix33 &mat, const Vector3 &translate = makeVector({0,0,0}));
 
+/// Transforms that can be used for ReconstructSurface
```

Comment:
This is a nice way to guide the use of a templated class while also providing a clear path to extending it.